packages/plugin-snowplow/lib/index.ts

```diff
@@ -45,9 +45,11 @@ export default class SnowplowBrowserPlugin extends PluginBase {
   }
 
   track(userId: string | undefined, event: Event) {
+    const slug = event.slug || event.name;
```

Comment:
Should we slugize the event name for the schema id?
`const slug = event.slug || _.kebabCase(event.name);`

Reply:
Good point, I targeted this fallback scenario mostly for the case when a user has events that already have snowplow compatible kebab case event names, so additional modification is not required. Also, I believe we should minimize additional dependencies or even own implementation of slugify function since it, probably, will miss some non-trivial cases.

Reply:
Yep, agree we should avoid extra dependencies. [Lodash is modular so we could just add the kebabcase](https://www.npmjs.com/package/lodash.kebabcase) rather than the whole lib and not have to worry about edge cases in our own regex.

Reply:
One thing I like about sticking with the provided values as-is is that there isn't anything we're doing that might be implicit/unpredictable to the user. `_.kebabCase()` could result in the same value for two events with similar names. As is, we'll use `name` (and potentially fail fast if the name isn't compatible in Snowplow) or `slug` if the user wants to provide their own custom value for Snowplow (or other plugins that use `slug` instead of `name`).